cli/src/wallet_cli/mod.rs

```diff
@@ -1292,7 +1394,16 @@ pub async fn prompt_internal(
                             ensure_password(wallet).await?;
                             allot_mana_command(wallet, mana, account_id).await
                         }
+                        WalletCommand::AnnounceCandidacy { account_id } => {
```

Comment:
I believe you have to call `ensure_password(wallet).await?;` before all operations that sends something

Reply:
done